mediachain/datastore/rocks.py

```diff
@@ -0,0 +1,51 @@
+import rocksdb
+import cbor
+from multihash import encode as multihash_encode, SHA2_256
+from base58 import b58encode
+
+
+class RocksDatastore(object):
+    def __init__(self, db_path):
+        opts = rocksdb.Options(create_if_missing=True)
+        self.db = rocksdb.DB(db_path, opts)
+
+    def get(self, ref):
+        try:
+            ref = ref.multihash_base58()
+        except AttributeError:
+            pass
+
+        ref = bytes(ref)
+        byte_string = self.db.get(ref)
+        try:
+            return cbor.loads(byte_string)
+        except ValueError:
+            return byte_string
+
+    def put(self, data_object, ref=None):
```

Comment:
That's true; I think I added the ref key as a paranoid guard against the transactor changing its multihash calculation or something, but that would break so many other things that it's silly to change this interface for that.  I'll fix that up when I circle back to this branch :)